src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -47,15 +47,16 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
 <body onload="scrollChat()">
 
   <nav>
-    <a id="navTitle" href="/">CodeU Chat App</a>
-    <a href="/conversations">Conversations</a>
-      <% if (request.getSession().getAttribute("user") != null) { %>
-    <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
-    <% } else { %>
-      <a href="/login">Login</a>
-    <% } %>
-    <a href="/about.jsp">About</a>
-  </nav>
+   <a id="navTitle" href="/">CodeU Chat App</a>
+   <a href="/conversations">Conversations</a>
+   <% if(request.getSession().getAttribute("user") != null){ %>
+     <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+   <% } else{ %>
+     <a href="/login">Login</a>
+     <a href="/register">Register</a>
```

Comment:
It looks like this whole block is shifted left by one column. Your IDE is doing something it shouldn't.